### ISSUE TITLE
The padding in rnn_trian.py 

### ISSUE THREAD

**Issue by xyx361100238** [jzi040941/PercepNet#13]
Dear Noah,
I was noticed the lines 23 & 24 in rnn_train.py, the final dimension of the data will keep the same, but the parameter  value must be  different.(I'm not sure if it will have an impact)
**nn.Conv1d** has parameter '**padding**' can solve this problem,such as:
        self.conv1 = nn.Conv1d(128, 512, 5, stride=1, padding=2)
        self.conv2 = nn.Conv1d(512, 512, 3, stride=1, padding=1)

Tip: padding = (kernel_size - 1) / 2

**Comment by jzi040941** [comment]
Hi, @xyx361100238
Thanks for letting me know, I was looking for this since keras have 'same' padding option but I couldn't find one for pytorch before.
I changed code base on your suggestion!
Thanks again.
